thrift/from_wire.go

```diff
@@ -123,6 +123,15 @@ func valueFromWireMap(spec *compile.MapSpec, w wire.Map) (map[string]interface{}
 	return result, nil
 }
 
+func mapEnumValueToName(enumSpec *compile.EnumSpec, result int32) interface{} {
+	for _, item := range enumSpec.Items {
```

Comment:
I think the same issue happens in code generation in `thriftrw` as well -- @abhinav what do you think about setting up a mapping internally in `compile.EnumSpec` and having a method to return the `EnumItem` given a value?